foreman_ansible_core.gemspec

```diff
@@ -20,4 +20,5 @@ DESC
   s.add_development_dependency 'rubocop', '~> 0.52'
   s.add_dependency 'foreman-tasks-core', '~> 0.1'
   s.add_dependency 'foreman_remote_execution_core', '~> 1.1'
+  s.add_dependency 'net-ssh', '~> 4.0'
```

Comment:
is the version pinned to 4.x because 5 requires Ruby >= 2.2.6?

Reply:
Not really, I just looked at what we had in foreman-packaging (4.0.1) and tried to make it compatible

Reply:
I tested with 5.0.1 and all seems fine, perhaps >= 4.0 and < 6 would be fine?